src/redux/actions.js

```diff
@@ -0,0 +1,109 @@
+import types from './constants';
+
+const BOOKS_URL = 'http://localhost:4730/books';
+
+export function addDummyBook() {
+  return { type: types.ADD_DUMMY };
+}
+
+export function updateBook(book) {
+  return { type: types.UPDATE_BOOK, book };
+}
+
+// fetchBookList
+function fetchBookListPending() { return { type: types.FETCH_BOOK_LIST_PENDING } }
+function fetchBookListSuccess(books) { return { type: types.FETCH_BOOK_LIST_SUCCESS, books } }
+function fetchBookListError(error) { return { type: types.FETCH_BOOK_LIST_ERROR, error } }
+export function fetchBookList() {
```

Comment:
mixture of named functions and arrow functions  - I think as we were fixing slides, we tried to stick with arrow functions to reduce the friction

Reply:
I'm not sure if I understand this right. Are you saying, we should change all function definitions in `actions.js` to arrow functions? @kof